sites/ademe/src/backend/templates/social-networks/course-badges.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n static %}{% spaceless %}
+{% with full_page_url=SITE.web_url|add:page_url %}
+    {% blocktrans asvar shared_subject %}Follow a course online with MBD{% endblocktrans %}
+    {% blocktrans with title=page_title|safe url=full_page_url asvar shared_sentence %}I just enrolled to the course "{{ title }}" on MBD: {{ url }}{% endblocktrans %}
+    <div class="social-network-badges">
+        <a
+            href="https://www.facebook.com/share.php?u={{ full_page_url }}"
+            class="social-network-badges__item"
+            target="_blank"
```

Comment:
For security purpose, it's common to complete a link with `target="_blank"` attribute with rel attribute sets to `noopener noreferrer`
```suggestion
            target="_blank"
            rel="noopener noreferrer"
```